shared/constants.py

```diff
@@ -8,24 +8,37 @@
 GOVERNANCE_AGENT_URL = os.getenv("ACAPY_GOVERNANCE_AGENT_URL", f"{url}:3021")
 GOVERNANCE_AGENT_API_KEY = os.getenv("ACAPY_GOVERNANCE_AGENT_API_KEY", adminApiKey)
 
-GOVERNANCE_FASTAPI_ENDPOINT = os.getenv("GOVERNANCE_FASTAPI_ENDPOINT", f"{url}:8100")
+GOVERNANCE_FASTAPI_ENDPOINT = os.getenv(
+    "GOVERNANCE_FASTAPI_ENDPOINT", f"{url}:8200"
+)  # governance-ga-web
 GOVERNANCE_ACAPY_API_KEY = os.getenv("GOVERNANCE_ACAPY_API_KEY", adminApiKey)
 
-TENANT_FASTAPI_ENDPOINT = os.getenv("TENANT_FASTAPI_ENDPOINT", f"{url}:8100")
+TENANT_FASTAPI_ENDPOINT = os.getenv(
+    "TENANT_FASTAPI_ENDPOINT", f"{url}:8300"
+)  # governance-tenant-web
+TENANT_ADMIN_FASTAPI_ENDPOINT = os.getenv(
+    "TENANT_ADMIN_FASTAPI_ENDPOINT", f"{url}:8100"
+)  # governance-multitenant-web
 TENANT_ACAPY_API_KEY = os.getenv("TENANT_ACAPY_API_KEY", adminApiKey)
 
 TENANT_AGENT_URL = os.getenv("ACAPY_TENANT_AGENT_URL", f"{url}:4021")
 TENANT_AGENT_API_KEY = os.getenv("ACAPY_TENANT_AGENT_API_KEY", adminApiKey)
 
 TRUST_REGISTRY_URL = os.getenv("TRUST_REGISTRY_URL", f"{url}:8001")
+TRUST_REGISTRY_FASTAPI_ENDPOINT = os.getenv(
+    "TRUST_REGISTRY_FASTAPI_ENDPOINT", f"{url}:8400"
+)  # governance-trust-registry
+
 
 WEBHOOKS_URL = os.getenv("WEBHOOKS_URL", f"{url}:3010")
 WEBHOOKS_PUBSUB_URL = os.getenv("WEBHOOKS_PUBSUB_URL", f"ws://{host}:3010/pubsub")
 
 ACAPY_MULTITENANT_JWT_SECRET = os.getenv("ACAPY_MULTITENANT_JWT_SECRET", "jwtSecret")
 ACAPY_ENDORSER_ALIAS = os.getenv("ACAPY_ENDORSER_ALIAS", "endorser")
 
-CLOUDAPI_URL = os.getenv("CLOUDAPI_URL", f"{url}:8100")
```

Comment:
the CLOUDAPI_URL env var is only used once and is identical to TENANT_FASTAPI_ENDPOINT. So I refactored that out

NB: CLOUDAPI_URL also removed in docker-compose and no longer needed in charts